llvm/include/llvm/Support/RISCVAttributes.h

```diff
@@ -47,6 +48,15 @@ enum AtomicABI : unsigned {
 };
 } // namespace RISCVAtomicAbiTag
 
+namespace RISCVX3RegUse {
+enum X3RegUsage : unsigned {
+  UNKNOWN = 0,
+  GP = 0,
```

Comment:
Copy paste mistakes? Why all 0s?

Reply:
ugh, you are correct. I haven't tested this properly yet, since I'm not 100% sure what the correct way to add the attribute will be. But I'll up date these to have the proper values from the psABI.